### ISSUE TITLE
Is it possible to use a pi zero as the base station?

### ISSUE THREAD

**Issue by Cryaniptic** [jeanlemotan/esp32-cam-fpv#13]
Apologies in advanced im not to experienced with this kind of stuff but I wanted to make an rc car/tank. With the current unavailiablity of pi 4s I was wondering if it would be possible to use a pi zero that I have lying about. 

Additionally can data be sent back to the esp32? It would simplify my project if i could control the car with a controller or something plugged into the pi.

**Comment by jeanlemotan** [comment]
Yes, but you still need an USB wifi adapter. At least one. As far as I'm aware, the wifi on the PI0W doesn't support monitor mode so you need the TL-WN722N for that.
Yes, data can be sent back to the esp32. Communication is bi-directional. ATM the base station only sends a `config` packet with camera related stuff like resolution etc, but there is also a `data` packet with controller things. It's not handled in the esp32 side - that's something you'd have to do yourself.
